api/app/Models/PoolCandidateSearchRequest.php

```diff
@@ -29,6 +29,7 @@
  * @property Illuminate\Support\Carbon $updated_at
  * @property Illuminate\Support\Carbon $deleted_at
  * @property Illuminate\Support\Carbon $request_status_changed_at
+ * @property array $reason
```

Comment:
This is just a comment so not controlling anything, but small typo, the field is string

Reply:
oops! I started with an array for some reason and forgot to change this! 😛 

Fixed in [fix typo in property comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8408/commits/ae0a9c9a487e89f30780293bdb4e9f084a2a44ff)